addons/nuxeo-binary-metadata/src/main/java/org/nuxeo/binary/metadata/internals/MetadataMappingDescriptor.java

```diff
@@ -86,7 +91,19 @@ public String getBlobXPath() {
         return blobXPath;
     }
 
+    /**
+     * @deprecated since 11.1, use {@link #ignorePrefix()}
+     */
+    @Deprecated(since = "11.1")
     public Boolean getIgnorePrefix() {
-        return ignorePrefix;
+        return !Boolean.FALSE.equals(ignorePrefix);
+    }
+
+    public boolean ignorePrefix() {
+    	return Boolean.TRUE.equals(ignorePrefix);
```

Comment:
The new `ignorePrefix()` should do `!Boolean.FALSE.equals(ignorePrefix)` in order to return `true` if no value has been defined, therefore "defaulting" to `true`.

Reply:
👍